charts/gha-runner-scale-set/values.yaml

```diff
@@ -68,6 +68,12 @@ githubConfigSecret:
 #       key: ca.crt
 #   runnerMountPath: /usr/local/share/ca-certificates/
 
+## Container mode is is an object that provides out-of-box configuration
+## for dind and kubernetes mode. Template will be modified as documented under the
+## template object.
+##
+## If any customization is required for dind or kubernetes mode, containerMode should remain
+## empty, and configuration should be applied to the template.
```

Comment:
I think we have `containerMode: none` too as a variation instead of empty, right? I remember Francesco implemented that after some discussion

Reply:
We decided back then not to use name "none" for the `containerMode`, but rather to comment it out if it is not specified :relaxed: